recipes/meson-python/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "meson-python" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/meson_python-{{ version }}.tar.gz
+  sha256: 841d68a9ff26b7f75c8aab23008966abc52646846e6f1b50de55f8961834ed0f
+  patches:
+    - ninja-overdep.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - colorama
+    - meson >=0.60.0
+    - ninja
+    - pep621 >=0.3.0
+    - pip
+    - python >=3.7
+    - tomli >=1.0.0
+    - typing-extensions >=3.7.4
+  run:
+    - colorama
+    - meson >=0.60.0
+    - ninja
+    - pep621 >=0.3.0
+    - python >=3.7
+    - tomli >=1.0.0
+    - typing-extensions >=3.7.4
+
+test:
+  imports:
+    - mesonpy
+  commands:
+    - pip check
+    - pytest tests -vv
+  source_files:
+    - tests/
+  requires:
+    - {{ compiler('c') }}
+    - git
+    - gitpython
+    - patchelf  # [linux]
+    - pip
+    - pytest
+    - pytest-mock
+
+about:
+  home: https://github.com/FFY00/meson-python
+  doc_url: https://meson-python.readthedocs.io/en/latest/
+  summary: Meson Python build backend (PEP 517)
+  license: EUPL-1.2
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - awvwgk
```

Comment:
Can you add me too @awvwgk? SciPy builds will depend on `meson-python`, so I'm interested in helping maintain this package.

Reply:
Sure